### PR TITLE
Gutenberg Blocks styling

### PR DESCRIPTION
**Pull request by radu-themeisle**
Adds styling for Gutenberg Blocks, especially where blocks were styled by the theme and not the plugin

### CODE REVIEW

File: obfx_modules/gutenberg-blocks/build/style.css [hunk 1]
```diff
@@ -1,12 +1,849 @@
...
+    flex: 1;
+    padding: 20px 10px;
+    margin: 0 20px
+}
+
+.wp-block-orbitfox-pricing-table .raised {
+    box-shadow: 0 16px 38px -12px rgba(0, 0, 0, 0.56), 0 4px 25px 0 rgba(0, 0, 0, 0.12), 0 8px 10px -5px rgba(0, 0, 0, 0.2)
+}
+
+.wp-block-separator {
```
Comment (abaicus) [inline obfx_modules/gutenberg-blocks/build/style.css]: @radu-themeisle 
40px margin for the `.wp-block-separator` is a bit too much. Also, it's not ok to style it directly as this is rendered as a simple separator. It should be specifically selected for the pricing table by prefixing it properly `.wp-block-orbitfox-pricing-table .wp-block-separator`

I'd also remove the margin-bottom for the paragraphs used as properties in the tables. 

image

**Comment by selul** [comment]
@HardeepAsrani can you help radu moves this into the composer library as we no longer have the code here.

**Comment by HardeepAsrani** [comment]
@selul So are we adding styles to Blocks? Can you explain so I can add it?

**Comment by abaicus** [comment]
@HardeepAsrani @selul What needs to happen here?

My guess is the style should be moved to otter-blocks composer module, right?

**Comment by HardeepAsrani** [comment]
@abaicus Yea, that's what I'm not sure of what we are doing here. If we are adding styles to Gutenberg blocks then yes, here it should be done: http://github.com/Codeinwp/gutenberg-blocks

Also, @radu-themeisle you need to put your CSS in each block's style.scss (front-end styles) and editor.scss (editor styles), like this: https://github.com/Codeinwp/gutenberg-blocks/blob/master/blocks/plugin-card/editor.scss

And run `npm run dev` to start the watch process.

A couple of blocks have multiple styles, so make sure you style them properly.

Also, started this ticket here where Radu can move the modifications: https://github.com/Codeinwp/gutenberg-blocks/issues/4

**Comment by rodica-andronache** [comment]
@abaicus maybe you can help here move things further? 👍

**Comment by HardeepAsrani** [comment]
Not sure if this is still relevant?

**Comment by rodica-andronache** [comment]
@abaicus @radu-themeisle I'm not sure what's this for, but maybe somebody can clear it up, so we know if it's still necessary or should be closed :)

**Comment by radu-themeisle** [comment]
I believe this should be closed as it was supposed to add style to some predefined blocks overriding the theme, but we decided this is not the way to go and developed Otter.

**Pull request closed without merging**